packages/admin/admin/src/stack/Stack.tsx

```diff
@@ -57,7 +57,7 @@ export interface BreadcrumbItem {
     parentId: string;
     url: string;
     title: React.ReactNode;
-    invisible: boolean;
+    stateUrl?: string;
```

Comment:
What do you think about saving `location.search` instead? stateUrl isn't very telling

Reply:
> What do you think about saving location.search instead?

we _are_ saving `location.pathname + location.search`

> stateUrl isn't very telling

true.

- urlUpdatedFromLocation?
- locationUrl
- overrideUrl
- or a single url string and a boolean `updatedFromLocation: true`

Reply:
> we are saving location.pathname + location.search

Of course, but I'm assuming that `breadcrumb.url` is the same as `location.pathname`, so it would be alright to only save location.search as `search?: string`. Correct me if I'm wrong.

Reply:
> I'm assuming that breadcrumb.url is the same as location.pathname

This is correct for a Grid (where eg. filter is saved in search) but not for Tabs (where active tab is a sub-url)